Oracle/harvest.sql

```diff
@@ -84,7 +119,12 @@ INSERT INTO harvest(NETWORKID, NETWORK_NAME, DATAMARTID, DATAMART_NAME, DATAMART
   case when (select count(*) from prescribing) > 0 then current_date else null end REFRESH_PRESCRIBING_DATE,
   case when (select count(*) from pcornet_trial) > 0 then current_date else null end REFRESH_PCORNET_TRIAL_DATE,
   case when (select count(*) from death) > 0 then current_date else null end REFRESH_DEATH_DATE,
-  case when (select count(*) from death_cause) > 0 then current_date else null end REFRESH_DEATH_CAUSE_DATE
+  case when (select count(*) from death_cause) > 0 then current_date else null end REFRESH_DEATH_CAUSE_DATE,
```

Comment:
perhaps the harvest table should be built from the `cdm_status` table. For example, use the `cdm_status.last_update` column for the refresh dates.